app/assets/javascripts/SourceCodeGlower/ImageAnnotationGrid.js

```diff
@@ -50,50 +47,27 @@ ImageAnnotationGrid.prototype.process_grid = function() {
 
 ImageAnnotationGrid.prototype.draw_holders = function() {
   var annot_grid = this.get_annotation_grid();
-  var holder, annot_text_id, horiz_range, vert_range, holder_width,
-      holder_height, holder_left, holder_top;
-
-  // Edges of the image
-  var image_preview   = document.getElementById('image_preview');
-  var image_container = document.getElementById('image_container');
-  var codeviewer      = document.getElementById('codeviewer');
-
-  var top_edge    = image_preview.offsetTop + codeviewer.scrollTop;
-  var left_edge   = image_preview.offsetLeft + image_container.scrollLeft;
-  var right_edge  = image_preview.offsetLeft + image_container.scrollLeft + codeviewer.offsetWidth;
-  var bottom_edge = image_preview.offsetTop + codeviewer.scrollTop + codeviewer.offsetHeight;
 
   for (var i = 0; i < annot_grid.length; i++) {
     var grid_element = annot_grid[i];
 
-    annot_text_id = grid_element.id;
-    horiz_range   = grid_element.x_range;
-    vert_range    = grid_element.y_range;
-    holder = document.getElementById('annotation_holder_' + annot_text_id);
-
-    // Left offset of the holder
-    holder_left = image_preview.offsetLeft + parseInt(horiz_range.start, 10);
-    // Top offset of the holder
-    holder_top  = image_preview.offsetTop + parseInt(vert_range.start, 10);
-
-    holder_width  = parseInt(horiz_range.end, 10) - parseInt(horiz_range.start, 10);
-    holder_height = parseInt(vert_range.end, 10) - parseInt(vert_range.start, 10);
-
-    holder.style.left = Math.max(0, holder_left - left_edge) + image_preview.offsetLeft + 'px';
-    holder.style.top  = Math.max(0, holder_top - top_edge) + image_preview.offsetTop + 'px';
-
-    if (holder_left > right_edge || holder_top > bottom_edge || holder_left + holder_width < left_edge ||
-        holder_top + holder_height < top_edge) {
-      // Draw nothing, as holder is out of bounds of codeviewer
-      holder.style.display = 'none';
-    } else {
-      // Holder within codeviewer, draw as much of it as fits.
-      holder.style.display = 'block';
-      holder.style.width  = Math.min(Math.min(holder_width, (holder_left + holder_width) - left_edge),
-                                     right_edge - holder_left - VERTICAL_SCROLLBAR_COMPENSATION) + 'px';
-      holder.style.height = Math.min(Math.min(holder_height, (holder_top + holder_height) - top_edge),
-                                     bottom_edge - holder_top - HORIZONTAL_SCROLLBAR_COMPENSATION) + 'px';
-    }
+    var annot_text_id = grid_element.id;
```

Comment:
Identifier 'annot_text_id' is not in camel case.<br>Identifier 'grid_element' is not in camel case.